control/control.go

```diff
@@ -706,6 +712,10 @@ func (p *pluginControl) UnsubscribeDeps(id string) []serror.SnapError {
 }
 
 func (p *pluginControl) verifyPlugin(lp *loadedPlugin) error {
+	if lp.Details.Uri != nil {
+		// remote plugin
+		return nil
```

Comment:
Is there any way to check for a valid uri? If it's an invalid uri, we can return err here instead of nil.